src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -119,10 +122,12 @@ private void UpdateFooter()
         // Build
         private void eventSource_BuildStarted(object sender, BuildStartedEventArgs e)
         {
+            startTime = DateTime.Now;
```

Comment:
Technically, I think these should be UTCNow...I'm not kidding when I say I saw a bug at one point from someone who built in different time zones and saw errors as a result. It 100% shouldn't matter in this case, but just thought I'd mention it.

Reply:
I'd think about calling and storing [`Stopwatch.StartNew()`](https://learn.microsoft.com/dotnet/api/system.diagnostics.stopwatch.startnew) instead. It's designed to deal with exactly the monotonic-clock situation, though it generally has more accuracy than we care about for build scenarios (we care about seconds, and ticks/15ms timer accuracy are irrelevant).